CHANGELOG.md

```diff
@@ -18,7 +18,6 @@ All notable changes to this project will be documented in this file.
 * ### NI-DMM
   * #### Added
   * #### Changed
-    * `nidmm.Session()` no longer takes id_query parameter
```

Comment:
Merge problem? Will wait for conflict resolution.

Reply:
Need #713 merged into master before updating this branch will make sense.